### PR TITLE
Assertion for aggregateData

### PR DESCRIPTION
**Pull request by grewered**
I edited DESCRIPTION file, and I have some questions about aggregateData

### CODE REVIEW

File: R/aggregateData.R [hunk 1]
```diff
@@ -22,6 +22,7 @@ aggregateData <- function (dat, subunits, units, aggregatemissings = NULL, renam
                         recodedData = TRUE, suppressErr = FALSE, recodeErr = "mci", verbose = FALSE) {
```
Comment (grewered) [inline R/aggregateData.R:22]: line 21: should I test subunits and units for something?

**Comment by grewered** [comment]
grafik

I don't think this is an error I can fix, it's something with downloading qpdf or ghostscript package.

File: R/aggregateData.R [hunk 2]
```diff
@@ -22,6 +22,7 @@ aggregateData <- function (dat, subunits, units, aggregatemissings = NULL, renam
...
 
   if (!is.data.frame(dat)) stop ("'dat' must be a data.frame.")
```
Comment (sachseka) [inline R/aggregateData.R:24]: please replace this stop by assertdata.frame

File: R/aggregateData.R (hunk 1 above)
Comment (sachseka) [reply R/aggregateData.R:22]: yes, please. see prep2GADS function in GADS output branch

**Review by sachseka — CHANGES_REQUESTED** [review]

**Comment by sachseka** [comment]
Please pay attention to checks

**Comment by grewered** [comment]
> Please pay attention to checks

it says that units and/or subunits is apparently "character", but from the prep2GADS function I got, that it's supposed to be type "data frame". 
so should I check them for character instead? because I don't really understand the error (ubuntu-20.04: lines 91-238, especially lines 139 and following). maybe we can look at it together?

also, 2 checks fail because they don't find an `values` object, but there is no `values` object in the .R doc anywhere - only in the function `recodeData` (which I haven't touched so far) - the checks also fail in the master branch.

File: R/aggregateData.R [hunk 3]
```diff
@@ -21,7 +21,8 @@ makeInputAggregateData <- function (subunits, units, recodedData = TRUE) {
 aggregateData <- function (dat, subunits, units, aggregatemissings = NULL, rename = FALSE,
                         recodedData = TRUE, suppressErr = FALSE, recodeErr = "mci", verbose = FALSE) {
 
-  if (!is.data.frame(dat)) stop ("'dat' must be a data.frame.")
+  lapply(c(dat, subunits, units), checkmate::assert_data_frame)
```
Comment (sachseka) [inline R/aggregateData.R:24]: lapply(list(...))

**Review by sachseka — APPROVED** [review]

**Pull request merged**